### DISCUSSION TITLE
Error importing a file

### DISCUSSION THREAD

**Discussion by Shubbu03** [Cyfrin/foundry-full-course-cu#2047] [Q&A]
Hey guys! I'm currently on the "deploy with anvil" part and while i'm importing files from the "lib/forge-std" i'm facing the following issue 
```
Source "/foundry/simple-storage/script/lib/forge-std/Script.sol" not found: File import callback not supported
```
can anybody please help me resolve this

**Comment by EngrPips** [comment]
Hello @Shubbu03, use `lib/forge-std/Script.sol` instead of `"/foundry/simple-storage/script/lib/forge-std/Script.sol"` Let's see how that plays out.

**Comment by poodlepp** [comment]
Hello,maybe the lib/forge-std dosn't exists,in this case ,you can reinstall it.
```
forge install foundry-rs/forge-std —no-commit
```

**Reply by EngrPips** [reply]
Indeed
